### DISCUSSION TITLE
v19.14.3-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#125] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.14.3`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.14.3-3.0.1:** (2024-04-13)

- Updated Youtube from `19.12.3` to `19.14.3`
- Added German Localization @zSaaiq & @AlexInABox
- Added Tweak YouGroupSettings @Poomsmart
- Updated Submodules
    - YTABConfig
    - protobuf 
    - Return YouTube Dislikes 
    - YouTubeHeader 
    - YouPiP 
    - iSponsorBlock 
    - YTUHD
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.14.3-3.0.1'>v19.14.3-3.0.1 - YTLitePlus</a>.</em>

**Comment by AlexInABox** [comment]
@Balackburn are you sure the .ipa is supposed to be included with this release? seems like an accident tbh

**Comment by huynlx** [comment]
pip button in video player not work

**Reply by abheizthe** [reply]
still worked for me. v19.15.1-3.0.1

**Comment by Valdularo** [comment]
Signing to Google is saying it can confirm the safety of the device. Think there might be an issue with this now...

**Reply by Balackburn** [reply]
Some people have this problem, unfortunately I have no way of solving it, the only solution is to connect with uYouEnhanced, delete it and reinstall YTLitePlus you will be logged.

**Comment by abheizthe** [comment]
there's an error when running workflow (complete with error code 2)

**Reply by Balackburn** [reply]
Indeed will look into it

**Comment by W3issFM** [comment]
Changing app icon results in "Error: Failed to set alternate icon" message

**Reply by Balackburn** [reply]
> Changing app icon results in "Error: Failed to set alternate icon" message

Indeed, arichornlover added this feature but it doesn't work, I will look if I can fix it otherwise I will remove / disable it.

**Reply by arichornlover** [reply]
There’s 2 possible reasons.
1. icons not being declared in the app.
2. a tweak that injects the App Icons onto the app without the proper compatibility.

**Comment by CSummerChild** [comment]
When I run the workflow the iPA doesn't complete

**Reply by Balackburn** [reply]
Try again it's fixed now

**Reply by arichornlover** [reply]
Hey Balackburn, I recently updated YouTubeRebornPlus to compile just like uYouPlus/uYouEnhanced.
You want YTLitePlus to follow the same implementation?

**Reply by Balackburn** [reply]
> Hey Balackburn, I recently updated YouTubeRebornPlus to compile just like uYouPlus/uYouEnhanced.
> 
> You want YTLitePlus to follow the same implementation?

What exactly is the difference between?

**Reply by arichornlover** [reply]
Not really sure since I am not a jailbreak user.

**Reply by Balackburn** [reply]
> Not really sure since I am not a jailbreak user.

What do you mean by compiling like uyouenhanced?

**Reply by arichornlover** [reply]
It uses the same Makefile & buildapp.yml from uYouEnhanced now, that’s what I am referring to.

**Reply by Balackburn** [reply]
@arichornlover 

Could you remove completely the change app icon feature from YTLitePlus and keep it only in uYouEnhanced I feel like it bloated the project a little bit (too much files) and it's unusable for now.

**Comment by Valdularo** [comment]
Is that why some setting seem to have disappeared? Something has changed in this app and I don’t like it.

**Reply by Balackburn** [reply]
What feature have changed that you don't like? I didn't removed any features.
